code/modules/materials/definitions/solids/materials_solid_wood.dm

```diff
@@ -43,12 +43,14 @@
 	sound_manipulate = 'sound/foley/woodpickup1.ogg'
 	sound_dropped = 'sound/foley/wooddrop1.ogg'
 	compost_value = 0.2
+	temperature_burn_milestone_material = TRUE
 
 /decl/material/solid/organic/wood/fungal
 	name = "towercap"
 	uid = "solid_wood_fungal"
 	color = "#e6d8dd"
 	hardness = MAT_VALUE_FLEXIBLE + 10
+	temperature_burn_milestone_material = FALSE
```

Comment:
thought: `temperature_burn_milestone_material` could be a typepath instead to avoid inheritance causing issues? so if `src.type == src.temperature_burn_milestone_material` it'd be a milestone material